tequilapi/endpoints/connection.go

```diff
@@ -46,6 +47,10 @@ type connectionRequest struct {
 	// required: true
 	// example: 0x0000000000000000000000000000000000000002
 	ProviderID string `json:"providerId"`
+
+	// connect options
+	// required: false
+	ConnectOptions dto.ConnectOptions `json:"connectOptions,omitempty"`
```

Comment:
It's dont have to repeat `connectOptions` in `connectionRequest`. IMHO `options` would be enought in REST contract